### PR TITLE
SITES-21521: Update icon in app and UIX extensions

### PR DESCRIPTION
**Pull request by irenelagno**
## Description

https://experience-qa.adobe.com/?devMode=true&shell_source=workspace&workspace=sites21521&shell_ims=prod#/aem/generate-variations/

## Related Issue
https://jira.corp.adobe.com/browse/SITES-21521

## Related PRs

- https://git.corp.adobe.com/CQ/aem-headless-ui-commons/pull/921
- https://github.com/adobe/aem-uix-examples/pull/35
- https://git.corp.adobe.com/CQ/universal-editor-utils/pull/257

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,12 +17,6 @@ See: http://aem.live/docs/sidekick-generate-variations
     - `aio console workspace select`
 - Populate the `.env` file in the project root and fill it as shown [below](#env)
 
-### Local Development
```
Comment (irenelagno) [inline README.md:20]: As we discussed, this option doesn't work. I removed it to avoid confusion

File: web-src/src/components/MainSidePanel.js [hunk 2]
```diff
@@ -206,13 +206,13 @@ export function MainSidePanel(props) {
                 const sessionDate = new Date(session.name);
                 const groupingLabel = groupingLabelGenerator(sessionDate);
 
-                return (<>
+                return (<Item key={sessionDate}>
```
Comment (irenelagno) [inline web-src/src/components/MainSidePanel.js:209]: Fixed issue, I noticed during development. See my comment https://github.com/adobe/aem-genai-assistant/pull/240/files#r1602159973

File: README.md [hunk 3]
```diff
@@ -17,12 +17,6 @@ See: http://aem.live/docs/sidekick-generate-variations
...
-
-- `npm start` to start your local Dev server
```
Comment (slitviachenko) [inline README.md:22]: @irenelagno , Should we delete https://github.com/adobe/aem-genai-assistant/blob/SITES-21521/package.json#L7 as well ?

**Review by slitviachenko — CHANGES_REQUESTED** [review]
I have one question, https://github.com/adobe/aem-genai-assistant/pull/273#discussion_r1604066130

File: README.md (hunk 3 above)
Comment (irenelagno) [reply README.md:22]: I will do with my next PR! good catch

Comment (irenelagno) [reply README.md:22]: fixed

**Comment by irenelagno** [comment]
@slitviachenko updated package.json based on your comment

**Review by slitviachenko — APPROVED** [review]

**Pull request merged**